docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -78,7 +78,7 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , ... ] )
 ### DROP COLUMNS
 
 `ALTER TABLE DROP COLUMNS` statement drops mentioned columns from an existing table.
-Note that this statement is only supported with v2 tables.
+Note that this statement is only supported with Delta lake tables.
```

Comment:
There are some V2 interface to make it working, e.g., `TableCatalog`. I think we should better mention them instead of one specific implementation

Reply:
+1. V2 tables here refer to tables implemented using DataSource V2 API.

Reply:
Thank you for the contribution team, the customer's would like to better understand which tables are v2 tables so that they can be aware of which table formats support these operations